vscode/package.json

```diff
@@ -1,13 +1,26 @@
 {
   "name": "dataform",
-  "version": "0.0.1",
+  "categories": [
+    "Programming Languages"
+  ],
+  "publisher": "dataform",
+  "version": "0.0.3",
+  "icon": "dataform_logo.png",
+  "repository": {
+    "url": "https://github.com/dataform-co/dataform/tree/master/vscode"
+  },
   "engines": {
-    "vscode": "^1.45.0"
+    "vscode": "^1.48.0"
   },
   "activationEvents": [
     "workspaceContains:**/*.sqlx"
   ],
   "main": "extension.js",
+  "dependencies": {
+    "vscode-languageclient": "^6.1.3",
+    "vscode-languageserver": "^6.1.1",
+    "vscode-languageserver-textdocument": "^1.0.1"
+  },
   "contributes": {
     "commands": [
```

Comment:
Can we clean the commands a bit? I'd suggest:

title: "Compile project"
category: "Dataform"